core-services/prow/02_config/_config.yaml

```diff
@@ -293,15 +293,22 @@ tide:
   queries:
   - author: openshift-bot
     missingLabels:
-    - needs-rebase
     - do-not-merge/hold
-    - do-not-merge/work-in-progress
     - do-not-merge/invalid-owners-file
+    - do-not-merge/work-in-progress
+    - needs-rebase
```

Comment:
Changing Prow config does not seem like something that should be included in this PR

Reply:
Fixing the prow config 👍🏻